scripts/menu.sh

```diff
@@ -507,9 +514,7 @@ function configure_gke() {
   export GPU_PER_NODE=${GPU_PER_NODE:-1}
 
   # The type of node for the consumer node pools
-  export CONSUMER_MACHINE_TYPE=${CONSUMER_MACHINE_TYPE:-n1-highmem-2}
-
-  export_gpu_constants
+  export CONSUMER_MACHINE_TYPE=${CONSUMER_MACHINE_TYPE:-n1-standard-2}
```

Comment:
Are the ratios of resources on the standard nodes more similar to the demands of the consumer pods?

Reply:
Yeah exactly. the consumers use something like .3 of a CPU, so we were wasting a lot of memory.

I think the resource requirements were more appropriate once upon a time, but we've made improvement.s

Reply:
Awesome! That's a great improvement, then.